test/common/test_data.ml

```diff
@@ -12,6 +12,18 @@ module type S = sig
   val create: Fpath.t -> (t, error) result Lwt.t
 end
 
+let output_of_command ?(env = [||]) ?(input = (fun _ -> ())) command =
+  let ic, oc, ec = Unix.open_process_full command env in
```

Comment:
`git` and associated tools are not the path here, so I am not sure how this is supposed to work.